static/js/admin.js

```diff
@@ -759,13 +765,20 @@ function _setup_page() {
                 full_name: JSON.stringify(full_name.val()),
             };
 
+            if (owner.val() !== undefined && owner.val() !== "") {
+                url = "/json/bots/" + email;
```

Comment:
yes, definitely.  Might be worth adding lint checks for e.g.:

```
'/["] + email'
"/['] + email"
```

To catch issues like this.